File: Unidad 4/C#/U4programa10.cs
using System;

namespace U4programa10 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string[] paises = { "Perú", "Portugal", "México", "Brasil", "Israel", "Argentina" };


            double longitud_Promedio = CalcularLongitudPromedio(paises);
            Console.WriteLine("La longitud promedio de las palabras es: " + longitud_Promedio);
        }

        static double CalcularLongitudPromedio(string[] arreglo)
        {
            int total_Caracteres = 0;

            foreach (string palabra in arreglo)
            {
                total_Caracteres += palabra.Length;
            }

            return (double)total_Caracteres / arreglo.Length;
        }
    }
    }

File: Unidad 4/Java/U4programa10.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u4programa10;

/**
 *
 * @author DELL
 */
public class U4programa10 {

    public static void main(String[] args) {
        String[] paises = { "China", "España", "México", "Portugal", "Estados Unidos", "Suecia" };

        double longitudPromedio = CalcularLongitudPromedio(paises);
        System.out.println("Longitud Promedio de las Palabras: " + longitudPromedio);
    }

    static double CalcularLongitudPromedio(String[] arreglo) {
        int totalCaracteres = 0;

        for (String palabra : arreglo) {
            totalCaracteres += palabra.length();
        }

        return (double)totalCaracteres / arreglo.length;
    }
}
